app/models/invoice.rb

```diff
@@ -248,8 +249,50 @@ def void_invoice!
   def ensure_number
     return if number.present?
 
-    formatted_sequential_id = format('%03d', sequential_id)
+    if organization.document_numbering.to_s == 'per_customer'
+      customer_slug = "#{organization.document_number_prefix}-#{format('%03d', customer.sequential_id)}"
```

Comment:
can you put a `NOTE` comment to explain whats the format we wait here?

Reply:
Added